torch/onnx/_internal/exporter.py

```diff
@@ -921,9 +943,7 @@ def adapt_torch_outputs_to_onnx(
 
         Args:
             model_outputs: The PyTorch model outputs.
-            model_with_state_dict: The PyTorch model to get extra state from.
-                If not specified, the model used during export is used.
-                Required when :func:`enable_fake_mode` is used to extract real initializers as needed by the ONNX graph.
+            model: The Pytorch model
```

Comment:
Suggest documenting motivation of why this argument is needed.

I wasn't sure even based on removed comment. Since this is dealing with output instead of input.

Reply:
Reverting this back